.env.config

```diff
@@ -14,11 +14,11 @@ JWT_SECRET='shhhh super secret code here bro'
 ROOT_URL='http://localhost:3000'
 
 # Credentials for the admin user created at app initialization
-ADMIN_EMAIL='admin@hackmit.org'
+ADMIN_EMAIL='admin@example.org'
 ADMIN_PASS='party'
 
 # Used to send verification, registration, and confirmation emails
-EMAIL_CONTACT='HackMIT Team <hackmit.reg.system@gmail.com>'
+EMAIL_CONTACT='HackMIT Team <team@example.com>'
```

Comment:
Maybe change this to also remove "HackMIT" and replace it with something like "Hackathon Team"?